source/composeK.js

```diff
@@ -2,7 +2,6 @@ import chain from './chain';
 import compose from './compose';
 import map from './map';
 
-
```

Comment:
Please revert this change.

Reply:
Hi there, you mean the @@ -2,7 +2,6 @@ import chain from './chain'; part correct? I am looking at the source now and it is importing chain. If this is a git history thing, how does one remove this bit of change from the commit?

Reply:
Ahh, it was prettier auto-formatting then eslint restoring the single-quotes that triggered that change. I am going to reset everything to master and reapply all the changes then .